src/server/api/medias/index.js

```diff
@@ -307,6 +318,7 @@ router.get('/url', (req, res) => {
   const now = new Date();
   const start = Math.round(now.getTime() / 1000);
   const end = start + parseInt(VALIDITY);
+  clientIP = hashClientIP ? clientIP : null;
```

Comment:
@jackvanb Why not simplify this to:

`const clientIP = hashClientIP ? requestIp.getClientIp(req) : null;`

Then you don't need line 278

Reply:
Done